lib/main_common.pm

```diff
@@ -1087,6 +1087,8 @@ sub load_consoletests {
     loadtest "console/prepare_test_data";
     loadtest "console/consoletest_setup";
     loadtest 'console/integration_services' if is_hyperv || is_vmware;
+    # This is a temporary solution for enabling open-vm-tools tests, will use YAML job template instead.
```

Comment:
There will be a standalone open_vm_tools test suite in yaml job template ?

Reply:
For now, a lot of test modules are included in console tests. To save time and get rid of unnecessary test modules, we'll only keep integration service and open-vm-tools tests with textmode test suite in future.